cves/kernel/CVE-2015-7566.yml

```diff
@@ -167,8 +164,8 @@ autodiscoverable:
 
     The answer field should be boolean. In answer_note, please explain
     why you come to that conclusion.
-  note:
-  answer:
+  note: You can brute force (or check physically) for this check if it was accounted for. 
```

Comment:
The discussion of the bugzilla report says that a fuzzer was used to discover this vulnerability, so it is definitely accounted for